test/integration/push_int_test.go

```diff
@@ -113,7 +113,10 @@ func (suite *PushIntegrationTestSuite) TestCarlisle() {
 
 	// anonymous commit
 	wd := filepath.Join(cp.WorkDirectory(), namespace)
-	cp = ts.SpawnWithOpts(e2e.WithArgs("install", suite.extraPackage), e2e.WithWorkDirectory(wd))
+	cp = ts.SpawnWithOpts(e2e.WithArgs(
+		"install", suite.extraPackage),
+		e2e.WithWorkDirectory(wd),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"))
```

Comment:
This has caused so many issues that I created a story for that: https://www.pivotaltracker.com/story/show/176192765